client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDriverUri.java

```diff
@@ -193,6 +195,21 @@ public Properties getProperties()
         return properties;
     }
 
+    public Optional<String> getTimeZoneID()
+            throws SQLException
+    {
+        Optional<String> tzId = TIMEZONE_ID.getValue(properties);
+
+        if (tzId.isPresent()) {
+            List<String> ids = Arrays.asList(java.util.TimeZone.getAvailableIDs());
+            if (ids.contains(tzId.get())) {
+                return tzId;
+            }
+            throw new SQLException("Specified TimeZoneID is not supported");
```

Comment:
Use `ZoneId` for validation and then just return it
```java
if (timezone.isPresent()) {
   try {
     return ZoneId.of(timezone.get());
   }
   catch (DateTimeException | ZoneRulesException e) {
    throw new SQLException("Specified timezone is not supported: " + timezone.get(), e);
  }
}
```

Reply:
Also, we don't need to return Optional at all, since we can return `ZoneId.systemDefault()` here when parameter is not set.